pkg/integration/integration_test.go

```diff
@@ -15,5 +15,7 @@ type IntegrationTestSuite struct {
 }
 
 func TestIntegrationTestSuite(t *testing.T) {
-	suite.Run(t, new(IntegrationTestSuite))
+	if !testing.Short() {
```

Comment:
I'm not sure how much code we want to run in CI. We can't, for example, access CEDAR, but do we still want to be running a few integration tests using login, etc? Would require separate flags, but totally doable.